### ISSUE TITLE
Spyder 2 icon theme has missing icons

### ISSUE THREAD

**Issue by oscargus** [spyder-ide/spyder#9773]
### Issue Report Checklist

* [x] Searched the [issues page](https://github.com/spyder-ide/spyder/issues?q=is%3Aissue) for similar reports
* [ ] Read the relevant sections of the [Spyder Troubleshooting Guide](https://github.com/spyder-ide/spyder/wiki/Troubleshooting-Guide-and-FAQ) and followed its advice
* [x] Reproduced the issue after updating with ``conda update spyder`` (or ``pip``, if not using Anaconda)
* [x] Tried basic troubleshooting (if a bug/error)
    * [x] Restarted Spyder
    * [ ] Tried the other applicable steps from the Troubleshooting Guide
* [x] Completed the **Problem Description**, **Steps to Reproduce** and **Version** sections below

## Problem Description
There is no icon for "Symbol finder" when using the Spyder 2 icon theme. Maybe not a big issue, but maybe the lack of reports for this indicates that not that many people are using the Spyder 2 theme (re #3894)?

![image](https://user-images.githubusercontent.com/8114497/60767411-0338ee00-a0b8-11e9-80c6-0c41a7306c50.png)

![image](https://user-images.githubusercontent.com/8114497/60767415-13e96400-a0b8-11e9-9faf-a601183ba6d9.png)

What is missing is a file called symbol_find.png as far as I can tell.

### What steps reproduce the problem?

1. Use Spyder 2 icons

### What is the expected output? What do you see instead?
Probably an icon would be nicer.

## Versions

* Spyder version: 4.0.0.dev0 and 3.3.4

**Comment by ccordoba12** [comment]
The Spyder 2 icon theme is mainly used as a fallback when the Spyder 3 theme fails for one reason or another. I guess that's not the only icon missing, I'm sure there are a lot more.

You're welcome to fix this (if you want) because we don't have time for it right now.

**Comment by goanpeca** [comment]
I do not think we will be updating the Spyder2 theme anymore and suggest we remove it from Spyder 5. Closing @ccordoba12?

**Comment by ccordoba12** [comment]
We really need a fallback theme in case there are problems loading web fonts. I'd say we could remove the Spyder 2 theme when we switch to the SVG based KDE theme in Spyder 5 or 6.

**Comment by ccordoba12** [comment]
The Spyder 2 icons were removed in Spyder 6, so this no longer applies. Besides, we improved loading the QtAwesome theme, so this no longer should be a problem.
